stix_shifter_modules/msatp/stix_translation/query_constructor.py

```diff
@@ -399,10 +399,4 @@ def translate_pattern(pattern: Pattern, data_model_mapper, options):
     time_range = options['time_range']
     translated_dictionary = QueryStringPatternTranslator(pattern, data_model_mapper, time_range)
     translated_query = translated_dictionary.qualified_queries
-    if len(translated_query) > 1:
```

Comment:
It's because in the past if we had more than a single query, we created a union query from all the queries, and if we had only a single query we leave it as it (I thought that we have to retrieve only a single query ). Then I realized that the EXECUTE flow allows handling a list of queries that are returned from the query_constructor module so I removed that check.